packages/composites/crypto-volatility-index/schemas/env.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$id": "https://external-adapters.chainlinklabs.com/schemas/crypto-volatility-index-adapter.json",
+  "title": "@chainlink/crypto-volatility-index-adapter env var schema",
+  "required": ["RPC_URL"],
+  "type": "object",
+  "properties": {},
```

Comment:
```suggestion
 "properties": {
   "RPC_URL": {
      "type": "string",
      "format": "uri"
    }
    },
```

Reply:
Actually this might get pulled in by "$ref": "https://external-adapters.chainlinklabs.com/schemas/ea-reference-data-reader.json". Resolve if so.

Reply:
This is already included since it has json-rpc-adapter as a dep